scripts/generate_pipeline_files.py

```diff
@@ -0,0 +1,169 @@
+import itertools
+import json
+import os
+import pathlib
+
+import click
+import yaml
+
+# ignore yaml alias
+yaml.Dumper.ignore_aliases = lambda self, data: True
+CURRENT_PATH = pathlib.Path(__file__).resolve().parent
+PROJECT_ROOT = CURRENT_PATH.parent
+DATASETS_PATH = PROJECT_ROOT / "datasets"
+AIRFLOW_TEMPLATES_PATH = PROJECT_ROOT / "templates" / "airflow"
+
+
+@click.command()
+@click.option(
+    "--dataset_id",
+    prompt="Your dataset name",
+    required=True,
+    type=str,
+    help="Dataset Name or Category that your pipeline belongs to",
+)
+@click.option(
+    "--pipeline_id",
+    "-p",
+    prompt="Your pipeline name",
+    required=True,
+    type=str,
+    help="The name of your pipeline",
+)
+def create_pipeline(dataset_id: str, pipeline_id: str):
+    new_pipeline_path = f"{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id}/"
+    dir = os.path.dirname(new_pipeline_path)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+        click.echo(
+            f"{DATASETS_PATH}/{dataset_id}/pipelines/{pipeline_id} has been created"
+        )
+    else:
+        click.echo(f"{new_pipeline_path} already exists")
+    create_dataset_yaml(dir, dataset_id)
+    create_pipeline_yaml(dir, dataset_id)
+
+
+def create_dataset_yaml(dir: str, dataset_id: str):
+    sample_yaml = None
+    output = {}
+    with open(f"{PROJECT_ROOT}/samples/dataset.yaml", "r") as sample_dataset_yaml:
```

Comment:
Same as the other comment. Use pathlib to help decrease the indentation

Reply:
updated here: https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/412/commits/9080e14727d66ba4533b6d3dfb8bf577b2ce5939#diff-b760de9bd0fdeea47e69e074deb04cb2ef5604a0ae70ef0fa58b04121d206318R65